controllers/tools/applications.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const applicationService = require('../../services/applications');
+const { purifyUserInput } = require('../../modules/validators');
+
+function init({ router }) {
+    router.route('/applications/:formId/:applicationId?').get(async (req, res, next) => {
+        try {
+            const formId = purifyUserInput(req.params.formId);
+
+            const applications = await applicationService.getApplicationsByForm(formId);
+            if (!applications) {
+                return next();
+            }
+            // via https://stackoverflow.com/questions/196972/convert-string-to-title-case-with-javascript#comment85199999_46959528
```

Comment:
Oh, I thought you might end up storing the title of the form in the database when saving an application. But this works too.